compiler/src/iree/compiler/Codegen/LLVMGPU/TransformExtensions/LLVMGPUExtensionsOps.td

```diff
@@ -682,4 +682,30 @@ def PackSharedMemoryAllocOp :  Op<Transform_Dialect, "iree.pack_shared_memory_al
   }];
 }
 
+def CreateMatmulMfmaTileSizesOp :
+  Op<Transform_Dialect, "iree.create_matmul_mfma_tile_sizes",
+    [MemoryEffectsOpInterface,
+     TransformOpInterface,
+     TransformEachOpTrait,
+     ParamProducerTransformOpTrait]> {
+  let description = [{
+    Create param of tile sizes based on the matmul sizes.
```

Comment:
I would add a warning to the description here that this is specifically for ROCm + GFX90a. Additionally add a note
```
This operation always succeeds and produces two handles to the two tile sizes.
```

Reply:
Edit: Actually this doesn't always succeed, enumerate the possible failure cases in the op description.

Reply:
minor-nit: The typical language for these ops is to say `This operation emits a definite failure if ...` but upstream is stricter about it than IREE is. Documentation of IREE transform ops is a bit sparse today.